### PR TITLE
DDF-4759 updated script to automate product site

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

replaces manual updating of product website `index`, `download`, and `documentation-versions` pages with scripting.

#### Select relevant component teams: 
@codice/docs 
@codice/website 

#### Ask 2 committers to review/merge the PR and tag them here.
@ahoffer
@brjeter
@clockard
@djblue
@jrnorth
@lessarderic
@ricklarsen - Documentation
@shaundmorris

#### How should this be tested?
load site into browser. 
verify content and formatting on these pages
- index
- download
- documentation-version

#### Any background context you want to provide?
This script will not update the documentation artifacts themselves.
Some older versions will remain hard-coded as they were maintained in a different repository 

#### What are the relevant tickets?
#4759 

#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: js/script.js [hunk 1]
```diff
@@ -2,15 +2,37 @@ var getLatestStableVersion = function(callback) {
...
+            var max = "0";
+            var currentVersion = "0";
+
+            data.forEach(element => {
+                var vers = element.tag_name
+                if (vers > max) {
+                    max = vers;
+                    currentVersion = element;
+                } else {
+                }
```
[the remark this answers is not in the document]
Comment (ricklarsen) [reply js/script.js]: thanks. fixed

File: js/script.js [hunk 2]
```diff
@@ -2,15 +2,37 @@ var getLatestStableVersion = function(callback) {
     var versionContent = $("#stable-version");
 
     $.ajax({
-        url: "https://api.github.com/repos/codice/ddf/releases/latest",
+        url: "https://api.github.com/repos/codice/ddf/releases",
         success: function(data) {
-            var dateStr = data.published_at.slice(0, 10);
-            var html = "<a href='" + data.html_url + "' target='_blank'>" + data.tag_name + "</a>" +
+            var len = data.length;
```
[the remark this answers is not in the document]
Comment (ricklarsen) [reply js/script.js]: removed

File: js/script.js [hunk 3]
```diff
@@ -2,15 +2,37 @@ var getLatestStableVersion = function(callback) {
...
+            var max = "0";
+            var currentVersion = "0";
+
+            data.forEach(element => {
```
[the remark this answers is not in the document]
Comment (ricklarsen) [reply js/script.js]: 👍

File: js/script.js [hunk 4]
```diff
@@ -2,15 +2,37 @@ var getLatestStableVersion = function(callback) {
...
+
+            var dateStr = currentVersion.published_at.slice(0, 10);
+            var html = "<a href='" + currentVersion.html_url + "' target='_blank'>" + currentVersion.tag_name.toUpperCase() + "</a>" +
                 " released on " + dateStr;
             versionContent.html(html);
             
+            var downloadButton = $("#download-button"), buttonHref, buttonHtml;
+
+            buttonHref = currentVersion.html_url
+            buttonHtml = "Download .zip <span class='glyphicon glyphicon-download'></span>";
```
[the remark this answers is not in the document]
Comment (ricklarsen) [reply js/script.js]: fixed

File: js/script.js [hunk 5]
```diff
@@ -24,30 +46,49 @@ var getLatestStableVersion = function(callback) {
     });
 };
 
-var updateDownloadButton = function(data) {
-    var downloadButton = $("#download-button"), href, html;
-    
+var setNewsAndEvents = function(data) {
+    var versionHistory = $("#version-history"),href,dateString,versionName;
+    data = data.reverse()
```
[the remark this answers is not in the document]
Comment (ricklarsen) [reply js/script.js]: fixed

File: js/script.js [hunk 6]
```diff
@@ -24,30 +46,49 @@ var getLatestStableVersion = function(callback) {
     });
 };
 
-var updateDownloadButton = function(data) {
-    var downloadButton = $("#download-button"), href, html;
-    
+var setNewsAndEvents = function(data) {
+    var versionHistory = $("#version-history"),href,dateString,versionName;
```
[the remark this answers is not in the document]
Comment (ricklarsen) [reply js/script.js]: done

File: js/script.js [hunk 7]
```diff
@@ -24,30 +46,49 @@ var getLatestStableVersion = function(callback) {
...
-    } else {
-        href = "https://github.com/codice/ddf/releases/latest";
-        html = "Download from GitHub";
+        data.forEach(element => {
+
+            href = element.html_url
+            dateString = element.published_at.slice(0, 10)
+            versionName = element.name
+
+            html = "<a class='list-group-item' href='" + href + "''>\n<h4 class='list-group-item-heading'>" + dateString + "</h4>\n<p class'list-group-item-text'>The DDF Development Team is pleased to announce the availability of " + versionName + "</p></a>"
```
[the remark this answers is not in the document]
Comment (ricklarsen) [reply js/script.js]: updated

**Comment by bdeining** [comment]
build now

**Pull request merged**